ci-operator/config/kubevirt/kubevirt-template-validator/kubevirt-kubevirt-template-validator-master.yaml

```diff
@@ -0,0 +1,50 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+images:
+- dockerfile_path: Dockerfile
+  to: kubevirt-template-validator
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.8"
+  namespace: ocp
+tests:
+- as: unittests
+  steps:
+    test:
+    - as: unittests
+      commands: go test -v ./...
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: e2e-functests
+  steps:
+    cluster_profile: gcp
+    test:
+    - as: e2e-functests
+      cli: latest
+      commands: |
+        ./hack/env/install-validator.sh
+        export component=template-validator-container
+        export VALIDATOR_IMAGE=$(echo $IMAGE_FORMAT | envsubst)
```

Comment:
The script `./hack/env/install-validator.sh` will use `$VALIDATOR_IMAGE`, so it needs to be called after the export.

Reply:
fixed